projects/1.16.1/assets/storage-tech/storagetech/lang/zh_cn.json

```
@@ -1,3 +1,108 @@
+  "item.storagetech.custom_storage_part.advanced": "%s 自定义存储元件",
+  "item.storagetech.custom_fluid_storage_part.advanced": "%s 自定义流体存储元件",
+  "item.storagetech.custom_energy_storage_part.advanced": "%s 自定义能量存储元件",
+  "item.storagetech.custom_chemical_storage_part.advanced": "%s 自定义化学品存储元件",
+  "block.storagetech.poedrive": "能量磁盘驱动器",
+  "block.storagetech.poeimporter": "能量输入总线",
+  "block.storagetech.poeexporter": "能量输出总线",
+  "block.storagetech.poefurnace": "集成电炉",
+  "block.storagetech.structureconstructor": "建构机",
+  "block.storagetech.chemicaldrive": "化学品磁盘驱动器",
+  "block.storagetech.chemicalimporter": "化学品输入总线",
+  "block.storagetech.chemicalexporter": "化学品输出总线",
+  "block.storagetech.chemicalgrid": "化学品终端",
+  "block.storagetech.partscombiner": "元件组合器",
+  "itemGroup.storagetech": "存储科技",
+  "gui.storagetech.poe_drive": "能量磁盘驱动器",
+  "gui.storagetech.poe_importer": "能量输入总线",
+  "gui.storagetech.poe_exporter": "能量输出总线",
+  "gui.storagetech.poe_furnace": "集成电炉",
+  "gui.storagetech.structure_constructor": "建构机",
+  "gui.storagetech.chemical_drive": "化学品磁盘驱动器",
+  "gui.storagetech.chemical_importer": "化学品输入总线",
+  "gui.storagetech.chemical_exporter": "化学品输出总线",
+  "gui.storagetech.chemical_grid": "化学品终端",
+  "gui.storagetech.chemical_filter": "化学品过滤器",
+  "gui.storagetech.partscombiner": "元件组合器",
+  "jei.storagetech.category.partscombiner": "元件组合器",
+  "sidebutton.storagetech.energy_restriction": "能量交换限制",
+  "sidebutton.storagetech.energy_restriction.true": "受限",
+  "sidebutton.storagetech.energy_restriction.false": "最大",
+  "sidebutton.storagetech.processingmode": "放置模式",
+  "sidebutton.storagetech.processingmode.true": "尝试跳过当前步骤",
+  "sidebutton.storagetech.processingmode.false": "上一步成功后才可继续",
+  "sidebutton.storagetech.pause": "状态",
+  "sidebutton.storagetech.pause.false": "暂停",
+  "sidebutton.storagetech.pause.true": "运行",
+  "sidebutton.storagetech.zero": "重置",
+  "sidebutton.storagetech.hologram": "范围投影",
+  "sidebutton.storagetech.hologram.false": "隐藏",
+  "sidebutton.storagetech.hologram.true": "显示",
+  "sidebutton.storagetech.viewchemicaltype": "种类",
+  "sidebutton.storagetech.viewchemicaltype.0": "所有",
+  "sidebutton.storagetech.viewchemicaltype.1": "气体",
+  "sidebutton.storagetech.viewchemicaltype.2": "灌注",
+  "sidebutton.storagetech.viewchemicaltype.3": "泥浆",
+  "sidebutton.storagetech.chemicaltype": "种类",
+  "sidebutton.storagetech.chemicaltype.0": "气体",
+  "sidebutton.storagetech.chemicaltype.1": "灌注",
+  "sidebutton.storagetech.chemicaltype.2": "泥浆",
+  "sidebutton.storagetech.sortingtype": "排序按",
+  "sidebutton.storagetech.sortingtype.0": "数量",
+  "sidebutton.storagetech.sortingtype.1": "名字",
+  "sidebutton.storagetech.sortingtype.2": "类型",
+  "sidebutton.storagetech.info": "信息",
+  "sidebutton.storagetech.info.false": "隐藏",
+  "sidebutton.storagetech.info.true": "显示",
+  "widget.storagetech.save": "保存",
+  "widget.storagetech.add": "增加",
+  "tooltip.storagetech.structurecopier.place": "在 %d %d %d 放置了 %s",
```

原文：
> Place %s at %d %d %d

其中`%s`和`%d`的顺序和目前翻译的顺序不同
格式符不是随便就可以调换位置的，按照目前语序需要加入位置标记：
```suggestion
  "tooltip.storagetech.structurecopier.place": "在 %2$d %3$d %4$d 放置了 %1$s",
```
下一行同样